src/app/personside/infotabs/varsel/Varsel.tsx

```diff
@@ -118,4 +144,11 @@ function Varsel({ varsel }: { varsel: VarselModell }) {
     );
 }
 
+function Varsel({ varsel }: { varsel: UnifiedVarsel }) {
+    if (isDittNavEvent(varsel)) {
+        return <DittNavEventVarsel varsel={varsel} />;
+    }
+    return <VarselVarsel varsel={varsel} />;
```

Comment:
`<VarselVarsel` ?

Reply:
Byttet navnene på komponentene ble;
* `DittNavEventVarsel` for varslene fra det nye systemet til dittnav
* `Varsel` som er varslene fra varsel-sløyfen
* `UnifiedVarsel` som samle komponent som bestemmer hvilken av de to ovenfor som skal brukes.